.github/workflows/ci.yml

```diff
@@ -449,5 +449,4 @@ jobs:
         run: |
           sleep 30s
           kubectl logs deployment.apps/consumer-deployment
-          kubectl logs deployment.apps/consumer-deployment | grep -q "consumed"
```

Comment:
Why are we removing these checks ? Any specific concern for failure.

Reply:
I changed the test case a little. Earlier the code was printing `consumed` keyword as soon as the pod start. Now it is not. So it fails to get that keyword.

Reply:
So we already have 30 seconds timeout above. We are removing checks then test will be successful always.
Should we increase timeout ?

Reply:
done